source/chapters/hardware.rst

```diff
@@ -170,6 +191,27 @@ audio interfaces have phono preamps on their deck inputs, but not necessarily
 on every deck input. Many higher-end all-in-one controllers also include audio
 interfaces with phono preamps.
 
+Driver Support
+^^^^^^^^^^^^^^
+Unlike some proprietary DJ programs, Mixxx works with any audio interface that
+your operating system has a driver to use — including for timecode vinyl (DVS)
```

Comment:
I suggest to replace

> [...] that your operating system has a driver to use

with 

> compatible with your operating system

You give more details in the next paragraphs, so here I think you can keep it more simple.